recipes/zlib/all/conandata.yml

```diff
@@ -1,15 +1,9 @@
 sources:
   "1.2.12":
-    url: [
-      "https://zlib.net/zlib-1.2.12.tar.gz",
-      # libpng does not provide zlib 1.2.12 yet.
-    ]
-    sha256: ""
```

Comment:
@uilianries It has been missed during review of https://github.com/conan-io/conan-center-index/pull/9980. Maybe add a hook?

Reply:
Oh, I see it's already implemented https://github.com/conan-io/hooks/pull/407 👍